src/unix/fs.c

```diff
@@ -457,7 +463,7 @@ static ssize_t uv__fs_preadv(uv_file fd,
 
 
 static ssize_t uv__fs_read(uv_fs_t* req) {
-#if defined(__linux__)
+#if defined(RETRY_PREADV_ON_LINUX)
```

Comment:
Here and in the next ones
```suggestion
#if TRY_PREADV
```